### ISSUE TITLE
Adding "profile shortcuts" to launchers?

### ISSUE THREAD

**Issue by KopfKrieg** [tuskyapp/Tusky#3220]
I'm using Tusky with multiple profiles, and while it's rather easy to switch between profiles within the app, I also would like to have the ability to do that from my launcher.

So, on my launcher (or what's considered the "desktop" on Android) I'd then have little shortcuts to launch Tusky with a specific profile, without having to switch profiles while in the app.

Most chat apps have something like that (launch $app with $contact as starting point, withoung having to specify a contact from within the app). Maybe it's possible to add a feature like this to Tusky?

* * * *
- Tusky Version: N/A
- Android Version: 13
- Android Device: Google Pixel 6a
- Mastodon instance (if applicable): All ;)

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
We already have shortcuts for posting with a certain account, they can be dragged onto launchers as well.

**Comment by KopfKrieg** [comment]
Took me a while to figure out *how*¹ I can create such a shortcut, but yes, that's what I mean. Except that I don't want to open the "create post" screen, but just the default timeline with the specific profile.




1\) For anyone else: Long press of the Tusky icon on your launcher, then a small pop-up should show your accounts. Those accounts can be dragged to a free space on the launcher and voilà, you've got a shortcut

**Comment by connyduck** [comment]
Yes it would be a nice feature to have other shortcuts as well. Any ideas where to integrate them and how to distinguish them from the existing ones?

**Comment by KopfKrieg** [comment]
Maybe in the profile settings, e.g. at the bottom with a button to create a launcher shortcut?

Links to specific tabs would also be cool, but no idea if it's possible to implement them in the current code base without too many changes, and also no idea where to place the "create shortcut" button then.
